aten/src/ATen/native/quantized/Copy.cpp

```diff
@@ -22,11 +27,19 @@ Tensor& quantized_copy_from_float_cpu_(Tensor& self, const Tensor& src) {
       self.device().type() == kCPU,
       "Quantized copy only works with QuantizedCPU Tensors");
   AT_DISPATCH_QINT_TYPES(self.scalar_type(), "Copy", [&]() {
-    float* src_data = src.data_ptr<float>();
-    scalar_t* self_data = self.data_ptr<scalar_t>();
-    for (const auto i : c10::irange(self.numel())) {
-      self_data[i] = quantize_val<scalar_t>(
-          self.q_scale(), self.q_zero_point(), src_data[i]);
+    if (self.qscheme() == kPerChannelAffine) {
+      quantize_tensor_per_channel_affine(src, self, self.q_per_channel_scales(),
+                                         self.q_per_channel_zero_points(),
+                                         self.q_per_channel_axis());
+    } else {
+      float* src_data = src.data_ptr<float>();
+      scalar_t* self_data = self.data_ptr<scalar_t>();
+      for (const auto i : c10::irange(self.numel())) {
+        auto sc = self.q_scale();
+        auto test = self.q_zero_point();
+        self_data[i] = quantize_val<scalar_t>(
+            self.q_scale(), self.q_zero_point(), src_data[i]);
```

Comment:
can we call the quantize_tensor_per_tensor_affine here as well? (can be a separate PR)